docs/workflows-run-test-cases.md

```diff
@@ -0,0 +1,120 @@
+# Test K3s, Kubernetes (Kubeadm) and MicroK8s
```

Comment:
This looks great. Thanks for the write up.

Reply:
You're welcome. Thank you!